app/static/js/trip_planner.js

```diff
@@ -7,44 +7,65 @@ var map;
 var rboxer = new RouteBoxer();
 var mapData = []
 var markers = [];
+var coords;
 
 $(document).ready(function () {
     launchMap();
 });
 
 function launchMap() {
+    var geoOptions = { maximumAge: 30000,  //  Valid for 5 minutes
+        timeout:5000,  // Wait 5 seconds
+        enableHighAccuracy:true
+    }
+
```

Comment:
This looks a whole lot like a corresponding section in report_map.js...We should refactor to reduce the code duplication here. Note that this comment applies to much of the content in this file.